### PR TITLE
Updates installation section

### PR DESCRIPTION
**Pull request by butschster**

### CODE REVIEW

File: docs/en/intro/install.md [hunk 1]
```diff
@@ -156,174 +169,259 @@ use Cycle\Annotated\Annotation\Column;
...
+    new Annotated\MergeColumns(),                   // Integrate @Table column annotations
+    new Schema\Generator\GenerateRelations(),       // Define entity relationships
+    new Schema\Generator\GenerateModifiers(),       // Apply schema modifications
+    new Schema\Generator\ValidateEntities(),        // Ensure entity schemas adhere to conventions
+    new Schema\Generator\RenderTables(),            // Create table schemas
+    new Schema\Generator\RenderRelations(),         // Establish keys and indexes for relationships
+    new Schema\Generator\RenderModifiers(),         // Implement schema modifications
+    new Annotated\MergeIndexes(),                   // Merge @Table index annotations
+    new Schema\Generator\SyncTables(),              // Align table changes with the database
+    new Schema\Generator\GenerateTypecast(),        // Typecast non-string columns
```
Comment (roxblnfk) [inline docs/en/intro/install.md]: Should we use attributes there instead of annotations?

**Review by roxblnfk — APPROVED** [review]

File: docs/en/intro/install.md (hunk 1 above)
Comment (roxblnfk) [reply docs/en/intro/install.md]: I mean `@Table`, `@Column`

Comment (butschster) [reply docs/en/intro/install.md]: + will fix it

File: docs/en/intro/install.md [hunk 2]
```diff
@@ -156,174 +169,259 @@ use Cycle\Annotated\Annotation\Column;
...
+    new Annotated\MergeColumns(),                   // Integrate #[Table] column annotations
+    new Schema\Generator\GenerateRelations(),       // Define entity relationships
+    new Schema\Generator\GenerateModifiers(),       // Apply schema modifications
+    new Schema\Generator\ValidateEntities(),        // Ensure entity schemas adhere to conventions
+    new Schema\Generator\RenderTables(),            // Create table schemas
+    new Schema\Generator\RenderRelations(),         // Establish keys and indexes for relationships
+    new Schema\Generator\RenderModifiers(),         // Implement schema modifications
+    new Annotated\MergeIndexes(),                   // Merge #[Table] index annotations
+    new Schema\Generator\SyncTables(),              // Align table changes with the database
+    new Schema\Generator\GenerateTypecast(),        // Typecast non-string columns
```
Comment (roxblnfk) [inline docs/en/intro/install.md]:
```suggestion
    new Schema\Generator\ResetTables(),             // Reconfigure table schemas (deletes columns if necessary)
    new Annotated\Embeddings($classLocator),        // Recognize embeddable entities
    new Annotated\Entities($classLocator),          // Identify attributed entities
    new Annotated\TableInheritance(),               // Setup Single Table or Joined Table Inheritance
    new Annotated\MergeColumns(),                   // Integrate table #[Column] attributes
    new Schema\Generator\GenerateRelations(),       // Define entity relationships
    new Schema\Generator\GenerateModifiers(),       // Apply schema modifications
    new Schema\Generator\ValidateEntities(),        // Ensure entity schemas adhere to conventions
    new Schema\Generator\RenderTables(),            // Create table schemas
    new Schema\Generator\RenderRelations(),         // Establish keys and indexes for relationships
    new Schema\Generator\RenderModifiers(),         // Implement schema modifications
    new Annotated\MergeIndexes(),                   // Merge table index attributes
    new Schema\Generator\SyncTables(),              // Align table changes with the database
    new Schema\Generator\GenerateTypecast(),        // Typecast non-string columns
```

**Pull request merged**